deploy/kubernetes/helm/che/custom-charts/che-jaeger/values.yaml

```diff
@@ -7,8 +7,12 @@
 # SPDX-License-Identifier: EPL-2.0
 #
 
-# Default values for postgres.
+# Default values for Jaegar
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 
+global:
+  ingress:
+    class: "nginx"
```

Comment:
I am not sure if it is needed since the field is introduced in deploy/kubernetes/helm/che/values.yaml